packages/patternfly-4/react-charts/src/components/ChartUtils/chart-interactive-legend.ts

```diff
@@ -51,29 +43,15 @@ const getInteractiveLegendItems = ({ chartNames, omitIndex }: ChartInteractiveLe
   return result;
 };
 
-// Returns styles for interactive legend items
-export const getInteractiveLegendItemStyles = (hidden: boolean = false) => {
-  return !hidden ? {} : {
-    labels: {
-      fill: chart_color_black_500.value
-    },
-    symbol: {
-      fill: chart_color_black_500.value,
-      type: 'eyeSlash'
-    }
-  };
-};
```

Comment:
Why are these functions being refactored? What's wrong with leaving this here?

Reply:
Reverted function order